ingestion_tools/dataset_configs/10302.yaml

```diff
@@ -96,6 +96,27 @@ rawtilts:
       - chlamy_visual_proteomics/{run_name}/AreTomo/{run_name}_dose-filt.tlt
       - chlamy_visual_proteomics/{run_name}/AreTomo/{run_name}_dose-filt.xf
       - chlamy_visual_proteomics/{run_name}/ctffind4/ctfphaseflip_ctffind4.txt
+    parent_filters:
+      exclude:
+        run: &excluded_runs
+        - ^01122021_BrnoKrios_arctis_lam1_pos4$
+        - ^01122021_BrnoKrios_arctis_lam1_pos5$
+        - ^01122021_BrnoKrios_arctis_lam2_pos13$
+        - ^01122021_BrnoKrios_arctis_lam3_pos27$
+        - ^06042022_BrnoKrios_Arctis_grid7_Position_29$
+        - ^08042022_BrnoKrios_Arctis_grid5_gistest_Position_4$
+        - ^12052022_BrnoKrios_Arctis_grid_newGISc_Position_38$
+        - ^14042022_BrnoKrios_Arctis_grid5_Position_1$
+        - ^15042022_BrnoKrios_Arctis_grid9_Position_32$
+        - ^15042022_BrnoKrios_Arctis_grid9_Position_65$
+        - ^17072022_BrnoKrios_Arctis_p3ar_grid_Position_101$
+        - ^17072022_BrnoKrios_Arctis_p3ar_grid_Position_35$
+        - ^17072022_BrnoKrios_Arctis_p3ar_grid_Position_68$
+        - ^17072022_BrnoKrios_Arctis_p3ar_grid_Position_76$
+        - ^27042022_BrnoKrios_Arctis_grid9_hGIS_Position_13$
+        - ^27042022_BrnoKrios_Arctis_grid9_hGIS_Position_33$
+        - ^27042022_BrnoKrios_Arctis_grid9_hGIS_Position_44$
+        - ^27042022_BrnoKrios_Arctis_grid9_hGIS_Position_7$
```

Comment:
If we exclude tomograms, rawtilts, etc from being imported by filtering out invalid runs from each sub-type, **we're still processing runs that are invalid for this dataset**. That's (barely) acceptable for the moment since we don't write any per-run data as output right now, but if we started writing `run_metadata.json` files, this falls apart. It seems like adding support for multiple include/exclude regexes to SourceGlobFinder would let us skip these runs at the appropriate level instead of repeating this list for multiple child types.

Reply:
So would an alternative (but likely hacky/clunky and non-sustainable) solution be adapting our current `runs.sources.source_glob` fields / regexes to use negative lookahead regex and filter out all of these?

Reply:
Yeah, in theory we could hack this right now with a super gnarly value in `runs.sources.source_glob.match_regex` value. It's probably not a ton of work to support multiple in/exclude regexes in that finder though 🤷‍♀️

Reply:
I would lean towards the exclude over the regex as it is more cleaner with an exclude.